knowledge_repo/app/migrations/versions/009eafe4838f_add_support_for_external_authentication.py

```diff
@@ -15,29 +15,28 @@
 
 
 def upgrade():
-    # Migrate from `usernames` to `identifiers`
```

Comment:
i think we should keep this comment in here

Reply:
I felt it was somewhat obvious and the code blocks are somewhat disjoint now we introduced the `with` statement.